Owin.Security.Providers/BattleNet/BattleNetAuthenticationHandler.cs

```diff
@@ -27,6 +28,40 @@ public BattleNetAuthenticationHandler(HttpClient httpClient, ILogger logger)
 		{
 			_httpClient = httpClient;
 			_logger = logger;
+
+			switch (Options.Region)
```

Comment:
@KetaSwe Does this work for you? I get an exception right here, because the Options property is null at this stage...